### PR TITLE
[Feature #17227] axelor-quality Module - updates

### PR DESCRIPTION
**Pull request by psh-axelor**

### CODE REVIEW

File: axelor-quality/src/main/resources/domains/QualityPictures.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<domain-models xmlns="http://axelor.com/xml/ns/domain-models" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://axelor.com/xml/ns/domain-models http://axelor.com/xml/ns/domain-models/domain-models_5.0.xsd">
+
+	<module name="quality" package="com.axelor.apps.quality.db"/>
+
+	<entity name="QualityPictures">
+
+		<string name="name" title="Name"/>
+		<many-to-one name="content" title="File" ref="com.axelor.meta.db.MetaFile"/>
```
Comment (sos-axelor) [inline axelor-quality/src/main/resources/domains/QualityPictures.xml]: name of field must be metaFile not 'content'.

File: axelor-quality/src/main/resources/views/QualityPictures.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+			<panel name="detailsPanel" colSpan="6">
+				<field name="name" colSpan="12"/>
+				<field name="createDate" colSpan="6"/>
+				<field name="qualityMeasuringPoint" hidden="true"/>
+			</panel>
+		</panel>
+	</form>
+
+	<action-record name="action-quality-picture-record-on-new" model="com.axelor.apps.quality.db.QualityPictures">
+		<field name="qualityMeasuringPoint" expr="eval: _parent" if="_parent?._model == 'com.axelor.apps.quality.db.QualityMeasuringPoint'"/>
```
Comment (sos-axelor) [inline axelor-quality/src/main/resources/views/QualityPictures.xml]: What is the need to set qualityMeasuringPoint ?

Comment (psh-axelor) [reply axelor-quality/src/main/resources/views/QualityPictures.xml]: Mistake, not needed.

File: axelor-quality/src/main/resources/domains/QualityPictures.xml (hunk 1 above)
Comment (psh-axelor) [reply axelor-quality/src/main/resources/domains/QualityPictures.xml]: Updated

**Pull request merged**